certbot/certbot/_internal/log.py

```diff
@@ -264,7 +264,7 @@ class TempHandler(logging.StreamHandler):
 
     """
     def __init__(self) -> None:
-        self._workdir = tempfile.mkdtemp()
+        self._workdir = tempfile.mkdtemp(prefix="certbot_log")
```

Comment:
I completely agree it doesn't matter much, _but_ I found the random string of characters immediately following "log" a little difficult to parse. I also noticed a few of the programs on my systems use underscores rather than hyphens for these files so I have the ever so slightest preference for doing the same.
```suggestion
        self._workdir = tempfile.mkdtemp(prefix="certbot-log-")
```